Tests/SimpleHTTPTests/Encoder/MultipartFormDataEncoderTests.swift

```diff
@@ -0,0 +1,158 @@
+import XCTest
+@testable import SimpleHTTP
+
+class MultipartFormDataEncoderTests: XCTestCase {
+
+    let crlf = EncodingCharacters.crlf
+
+    func test_encode_multipartAddData_bodyPart() throws {
```

Comment:
I think the test is about single multipart? Could you make it more explicit? (test_encode_singleMultipart_returnEncoded for instance)